be/src/formats/parquet/stored_column_reader.cpp

```diff
@@ -265,6 +270,15 @@ void RepeatedStoredColumnReader::_delimit_rows(size_t* num_rows, size_t* num_lev
     for (; levels_pos < _levels_decoded && rows_read < *num_rows; ++levels_pos) {
         rows_read += _rep_levels[levels_pos] == 0;
     }
+    if (levels_pos < _levels_decoded) {
+        // means rows_read == *num_rows, but notice, ++levels_pos in for-loop will take one step forward, so we need -1
+        levels_pos--;
```

Comment:
Because we call `reset` in every `read_records`, so the `levels_pos` must be greater than zero?

Reply:
we can make sure `levels_pos >= 1` in this function.